index.js

```diff
@@ -0,0 +1,61 @@
+const { Bot } = require('grammy');
+
+const bot = new Bot('5532232117:AAG_8EgKVkKd04srN7B5aHLKatC3biacIHc');
+bot.use((ctx, next) => {
+    ctx.state = {};
+    return next();
+})
+
+const middlewareMatchToSplit = function (ctx, next){
+    if(ctx.match === ''){
+        return ctx.reply("Please enter a number")
+    }
+    ctx.state.arguments = ctx.match.split(' ');
+    const validateNumbers = ctx.state.arguments.some((elem) => {
+        return Number.isNaN(Number(elem))
+    })
+
+    if(validateNumbers === true){
+        return ctx.reply("Incorrect number entered")
+    } else {
+        return next()
+    }
+}
+
+bot.command("dimagay", (ctx)=> ctx.reply("WHY ARE YOU GAy"));
+bot.command("start", (ctx) => ctx.reply("welcome! "));
+bot.command("stop", (ctx) => ctx.reply("Okay. I`m will sleep"));
+bot.command("add",middlewareMatchToSplit, (ctx) => {
+    let result = ctx.state.arguments.reduce((previousValue, currentValue) => {
```

Comment:
Why do you use let instead of const? I dont see any place where this value may be changed